istio-control/istio-discovery/templates/telemetryv2.yaml

```diff
@@ -122,5 +122,89 @@ spec:
                 code:
                   local:
                     inline_string: envoy.wasm.stats
+
+---
+{{- if .Values.telemetry.v2.sdfilter }}
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: stackdriver-filter
+  {{- if .Values.global.configRootNamespace }}
+  namespace: {{ .Values.global.configRootNamespace }}
+  {{- else }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+spec:
+  configPatches:
+    - applyTo: HTTP_FILTER
+      match:
+        context: SIDECAR_OUTBOUND
```

Comment:
This needs to use proxymatch for 1.4

Reply:
I would do the proxyMatch part in this PR: https://github.com/istio/installer/pull/584